lisa/tests/base.py

```diff
@@ -886,16 +902,32 @@ def test_dmesg(self, level='warn', facility=None) -> ResultBundle:
             able to print it, so specifying it may lead to no entry being
             inspected at all. If ``None``, the facility is ignored.
         :type facility: str or None
+
+        :param ignored_patterns: List of regexes to ignore some messages.
+        :type ignored_patterns: list or None
         """
         levels = DmesgCollector.LOG_LEVELS
         # Consider as an issue all levels more critical than `level`
         issue_levels = levels[:levels.index(level) + 1]
+
+        logger = self.get_logger()
+
+        if ignored_patterns:
+            logger.info('Will ignore patterns in dmesg output: {}'.format(ignored_patterns))
+            regex_whitelist = [
+                re.compile(pattern)
+                for pattern in ignored_patterns
+            ]
+        else:
+            regex_whitelist = []
```

Comment:
Nit: might want to rename that to `regex_ignore` for consistency. Otherwise LGTM.